brainiak/fcma/classifier.py

```diff
@@ -502,3 +528,35 @@ def decision_function(self, X=None):
             self.test_data_ = self._prepare_test_data(normalized_corr_data)
         confidence = self.clf.decision_function(self.test_data_)
         return confidence
+
+    def score(self, X, y, sample_weight=None):
+        """ returns the mean accuracy on the given test data and labels
+
+        NOTE: This function will not work in sklearn.svm.SVC with precomputed
+        kernel when the kernel matrix is computed portion by portion.
+
+        Parameters
+        ----------
+        X: a list of numpy array in shape [num_TRs, self.num_voxels\_]
+            Test samples.
+            It is not used in sklearn.svm.SVC with precomputed
+            kernel when the kernel matrix is computed portion by portion.
```

Comment:
You say at the beginning of the docstring that the function does not work in this case. Here you say `X` is not used, implying the function does work. Can you please clarify?